pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -36,6 +36,9 @@ stdenv.mkDerivation {
   # required to support pthread_cancel()
   NIX_LDFLAGS = stdenv.lib.optionalString (!stdenv.isDarwin) "-lgcc_s";
 
+  # without this, git fails when trying to check for /etc/gitconfig existence
+  __propagatedSandboxProfile = stdenv.lib.sandbox.allowDirectoryList "/etc";
```

Comment:
`__` prefixes are used for builtins.  I would recommend against this naming convention, and suggest that you use a single `_` prefix if such prefixes are needed.